SharedProjects/GltfExport.Entities/GLTFMaterial.cs

```diff
@@ -8,6 +8,10 @@ namespace GLTFExport.Entities
     [DataContract]
     public class GLTFMaterial : GLTFIndexedChildRootProperty
     {
+        internal static float[] one2 = { 1.0f, 1.0f };
```

Comment:
can we instead add a definition to our Tools.Math file with definitions of vector2, vector3 and the zero and one constants, similar to babylon.js?

Reply:
Yes we may have this , but i recommend to introduce this vector class directly into the GltfExport.Entities to avoid new dependencies. The gltf project is supposed to be standalone and self sufficient (wich its not the case today). So, similar to babylon.js, introduce one Maths.cs file with Vector2 and Vector3, and some logic to replace the MathUtilities used in  only one place so far.

Reply:
That seems like an alright decision, I just like code reuse, and not making architectural decisions for convenience sake, Currently we take dependencies on some shared utilities, shared projects, as long as we don't take any additional dependencies directly from Max or/Maya2Babylon.

Ultimately, it would be a good goal to architect such that we can make Babylon2GLTF its own independent project.

Contributing to our utilities should still allow the project to be relatively standalone, right? That would be akin to using a shared math library or scientific computing library in my opinion.

Reply:
I definitely agree with adding a helper function to compare arrays via IsAllAlmostEqualTo,

Alternatively you can override IsAlmostEqualTo to use float[] as well.

Reply:
Ok this is what i done. I guess PR is ready now.